include/svnlook.php

```diff
@@ -808,8 +808,10 @@ function applyGeshi($path, $filename, $language, $rev, $peg = '', $return = fals
 		$this->geshi->set_header_type(GESHI_HEADER_NONE);
 		$this->geshi->set_overall_class('geshi');
 		$this->geshi->set_tab_width($this->repConfig->getExpandTabsBy());
-		$this->geshi->enable_line_numbers(GESHI_FANCY_LINE_NUMBERS);
-		$this->geshi->enable_ids(true);
+		if ($highlight == 'file') {
```

Comment:
Looks good to me, but please add a newline before IF. Makes things more readable. One before line 806 would make sense in my opinion as well.

Reply:
Added newline for readability. At line 800 and 806 as well. Just to be consistent.